subsys/bluetooth/controller/ll_sw/ull_llcp_pdu.c

```diff
@@ -901,8 +901,15 @@ void llcp_pdu_decode_cis_rsp(struct proc_ctx *ctx, struct pdu_data *pdu)
 		ctx->data.cis_create.cis_offset_max = cis_offset_max;
 	}
 
+	/* FIXME: We do not pick the instant from the response; if we do, then
+	 *        we need to calculate again a new offset because the ACL and
+	 *        ISO intervals can be dissimilar and hence will be a different
+	 *        CIS offset when sending the CIS_IND PDU at this instant.
+	 *
+
 	ctx->data.cis_create.conn_event_count =
 		sys_le16_to_cpu(pdu->llctrl.cis_rsp.conn_event_count);
+	 */
```

Comment:
@erbr-ot This change for the central to not pick the peripheral sent instant value is failing unit test; because `ull_central_iso_cis_offset_get()` is mocked to do nothing, leading to `conn_event_count` not being assigned the value to be verified.

Your review and ideas will be helpful here.